### PR TITLE
Added bulkTransfers and transactionRequests endpoints

### PR DESCRIPTION
**Pull request by rmothilal**
Transactions requests handler added
Added endpoints for bulk transfers
Updated dependencies
Bump version
Standard fixes

### CODE REVIEW

File: src/bulkTransfers/handler.js [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Murthy Kakarlamudi murthy@modusbox.com
```
Comment (elnyry) [inline src/bulkTransfers/handler.js]: Please take credit ;-)

Comment (rmothilal) [reply src/bulkTransfers/handler.js]: lol

File: src/bulkTransfers/routes.js [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Murthy Kakarlamudi murthy@modusbox.com
+ --------------
```
Comment (elnyry) [inline src/bulkTransfers/routes.js]: Same as earlier, credit..

Comment (rmothilal) [reply src/bulkTransfers/routes.js]: fixed, thanks

**Comment by elnyry** [comment]
@rmothilal 

Thanks for the PR in such short turn around time.. Really appreciate this.

It seems the 'extentionList' element is being rejected in a POST call, whereas it should be there..

```
2019-07-24T19:28:07.265Z - info: L1p-Trace-Id=undefined - Response: ValidationError: "extensionList" is not allowed
2019-07-24T19:28:50.721Z - info: L1p-Trace-Id=undefined - Method: post Path: /bulkTransfers Query: {}
2019-07-24T19:28:50.721Z - info: L1p-Trace-Id=undefined - Headers: {"accept":"application/vnd.interoperability.bulkTransfers+json;version=1","content-type":"application/vnd.interoperability.bulkTransfers+json;version=1.0","date":"Thu, 25 Jul 2019 10:22:12 GMT","fspiop-source":"payerfsp","fspiop-destination":"payeefsp","fspiop-uri":"/bulkTransfers","fspiop-http-method":"POST","user-agent":"PostmanRuntime/7.15.0","cache-control":"no-cache","postman-token":"e7b45dd2-4dba-417b-9404-57052ca58912","host":"samuels-macbook-pro-5497.local:8444","accept-encoding":"gzip, deflate","content-length":"2749","connection":"keep-alive"}
2019-07-24T19:28:50.728Z - info: IN Bulk Transfers POST:: received: post /bulkTransfers b51ec534-ee48-4575-b6a9-ead2955b8069.
2019-07-24T19:28:50.730Z - info: Executing PUT: [http://localhost:1080/bulkTransfers/b51ec534-ee48-4575-b6a9-ead2955b8069], HEADERS: [{"Content-Type":"application/vnd.interoperability.transfers+json;version=1.0","FSPIOP-Source":"payeefsp","FSPIOP-Destination":"payerfsp","Date":"Wed, 24 Jul 2019 19:28:50 GMT","FSPIOP-Signature":"{\"signature\":\"abcJjvNrkyK2KBieDUbGfhaBUn75aDUATNF4joqA8OLs4QgSD7i6EO8BIdy6Crph3LnXnTM20Ai1Z6nt0zliS_qPPLU9_vi6qLb15FOkl64DQs9hnfoGeo2tcjZJ88gm19uLY_s27AJqC1GH1B8E2emLrwQMDMikwQcYvXoyLrL7LL3CjaLMKdzR7KTcQi1tCK4sNg0noIQLpV3eA61kess\",\"protectedHeader\":\"eyJhbGciOiJSUzI1NiIsIkZTUElPUC1Tb3VyY2UiOiJwYXllZWZzcCIsIkZTUElPUC1EZXN0aW5hdGlvbiI6InBheWVyZnNwIiwiRlNQSU9QLVVSSSI6Ii9idWxrVHJhbnNmZXJzL3VuZGVmaW5lZCIsIkZTUElPUC1IVFRQLU1ldGhvZCI6IlBVVCIsIkRhdGUiOiIifQ\"}","FSPIOP-HTTP-Method":"PUT","FSPIOP-URI":"/bulkTransfers/b51ec534-ee48-4575-b6a9-ead2955b8069"}], BODY: [{"completedTimestamp":"2019-07-24T19:28:50.729Z","individualTransferResults":[{"transferId":"b51ec534-ee48-4575-b6a9-ead2955b8069","fulfilment":"XoSz1cL0tljJSCp_VtIYmPNw-zFUgGfbUqf69AagUzY"},{"transferId":"b51ec534-ee48-4575-b6a9-ead2955b8069","fulfilment":"XoSz1cL0tljJSCp_VtIYmPNw-zFUgGfbUqf69AagUzY"}],"bulkTransferState":"COMPLETED"}]
2019-07-24T19:28:50.744Z - error: request to http://localhost:1080/bulkTransfers/b51ec534-ee48-4575-b6a9-ead2955b8069 failed, reason: connect ECONNREFUSED 127.0.0.1:1080
2019-07-24T19:28:50.745Z - info: L1p-Trace-Id=undefined - Response: null Status: 202
```

When I remove the extensionList element and send the POST request, its fine..

Please look into it.

**Pull request merged**